app/controllers/api/accounts_controller.rb

```diff
@@ -6,23 +6,44 @@
 #  https://github.com/puzzle/cryptopus.
 
 class Api::AccountsController < ApiController
+  helper_method :team
 
+  # GET /api/api_users
   def index
     authorize Account
     accounts = current_user.accounts
     accounts = find_accounts(accounts)
     render_json accounts
   end
 
+  # GET /api/api_users/:id
   def show
-    account = Account.find(params[:id])
     authorize account
     account.decrypt(decrypted_team_password(account.group.team))
     render_json account
   end
 
+  # PATCH /api/api_users/:id?Query
+  def update
+    update_account
+    authorize account
+    account.update!(permitted_attributes(account))
```

Comment:
update! saves the records to the db !
you're triggering update and save !
like this all changes are persisted twice !
check api doku for #save and #update